homeassistant/components/kostal_plenticore/switch.py

```diff
@@ -44,6 +44,16 @@ class PlenticoreSwitchEntityDescription(SwitchEntityDescription):
         off_value="2",
         off_label="Automatic economical",
     ),
+    PlenticoreSwitchEntityDescription(
```

Comment:
So this is only available when using a service code? If someone didnt add a service code, do we still create this entity?

Reply:
The data is still available to get without a service code. Its just not settable without it.

Reply:
So we get a switch that does not work? Ideally we create a binary sensor for that then

Reply:
As in, create a switch if we can update it, create a binary sensor if we can only read it

Reply:
Maybe we should put the service charge into another pr? It would require introducing binary sensors to the component.

Reply:
The switch i added